src/content/docs/en/reference/components-reference.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: Built-in Components Reference
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 2
```

Comment:
Until we go through and structure this page and we decide what it will look like, temporary fix so that only the components show up in the TOC.

(The alternative would be to convert `###` to `<h3>`, which would also be fine! This way preserves our normal Markdown authoring.)